### ISSUE TITLE
Tab Order is not Persisted

### ISSUE THREAD

**Issue by Slartibartfast779** [sparrowwallet/sparrow#874]
Hi,
I like to have my wallet tabs in a particular order, starting with a hot wallet followed on the right by cold wallets. I have found the controls to move each of the tabs but after I log out and log back in, the tabs have rearranged themselves.
Is there a way, the position of the tabs could be saved please?  
Thanks in advance
S

**Comment by Giszmo** [comment]
Title should be "tab order is not persisted". Else, the issue reads like a duplicate of #476

**Comment by Slartibartfast779** [comment]
Thanks

**Comment by craigraw** [comment]
Fixed in c9d6bb350d77b14c7fb68f9b24f7b9576370dd64.
